modules/oadp-mcg-performance-tuning.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
Should this module contain a set of instructions? OR
Did you intend to create a `concept` or `reference` module?